README.md

```diff
@@ -122,6 +122,13 @@ console.log(micromatch.isMatch('foo', ['b*', 'f*'])) //=> true
 
 You can mix and match these features to create whatever patterns you need!
 
+## When you SHOULD NOT use micromatch?
+If you are trying to accomplish the same filtering that the `.gitIgnore` `git` does, this library is NOT going to accomplish 1-1 javascript transposition of the matching algorithm.
+
+This library is meant to be a functionally drop-in replacement for minimatch and multimatch and neither of them are conforming to https://man7.org/linux/man-pages/man7/glob.7.html.
```

Comment:
Remove this line, since the glob(3) linux man page is not a specification or idiomatic reference for `.gitignore`.